recipes/jupyterlab-tour/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "jupyterlab-tour" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a1fb867d5278f27dc43ffd8a976788e5a6fff769c7f482aed5f6c9aace9b47a1
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - nodejs >=12.0.0
```

Comment:
Actually, if there's a way to use the assets directly from the sdist (e.g. by deleting the `pyproject.toml`) that can reduce the chances of getting... surprises.

Reply:
skip-npm might suficient:

```
{{ PYTHON }} -m pip install --install-option="--skip-npm" . -vv
```

Reply:
Thanks for this great suggestion